### ISSUE TITLE
Super unit in tribesconfig

### ISSUE THREAD

**Issue by QuasiStellar** [GAIGResearch/Tribes#284]
https://github.com/GAIGResearch/Tribes/blob/master/src/core/TribesConfig.java#L104
It's called a giant. Special tribes have their own super units.

**Comment by GAIGResearch** [comment]
True. There's no planned work into allowing new units yet - this will come into play if we decide to do this.
